docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_advanced_arming:
+
+***********************************************
+Advanced Arming
+***********************************************
+Vertiq modules can support an advanced arming feature, allowing the user to control the armed state of the module with throttle commands or manually 
+and to configure specific behaviors to occur at armed state transitions. Modules will not react to throttle messages until they have armed, providing improved safety. 
+The configurable behaviors on armed state transitions allow users to easily integrate advanced behaviors into their setup just by controlling the throttle messages they send, 
+simplifying flight controller integration. 
+
+The :ref:`arming_module_support` section below details which Vertiq modules currently support advanced arming.
+
+.. _arming_module_support:
+
+Module Support
+===============
+
+Speed Modules
+**************
+
+.. table:: Module Support for Advanced Arming
+
+	+-------------+------------------------------------+
+	| Module      | Advanced Arming Support            |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support Advanced Arming features.
+
+.. table:: Servo Module Support for Advanced Arming
+
+	+-------------+------------------------------------+
+	| Module      | Advanced Arming Support            |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+Armed States
+===============
+Modules can be in one of two armed states at any time:
+
+* **Armed**: In this state the module will spin when it receives a :ref:`throttle command <manual_throttle>`, so it is not safe to be near a module with an attached propeller when the module is armed.

```

Comment:
In this state, ... . **It is unsafe to be near an armed module with an attached propeller.**